src/js/components/Card/stories/Media.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import {
+  Anchor,
+  Box,
+  Card,
+  Heading,
+  CardBody,
+  CardFooter,
+  Grommet,
+  Image,
+  Paragraph,
+} from 'grommet';
+
+const theme = {
+  global: {
+    font: {
+      family: `Comic Sans MS, -apple-system,
+         BlinkMacSystemFont, 
+         "Segoe UI", 
+         Roboto`,
+    },
+  },
+  card: {
+    elevation: 'none',
+    background: 'light-2',
+    footer: {
+      pad: 'medium',
+    },
+  },
+};
+
+const Example = () => (
+  <Grommet theme={theme}>
+    <Box pad="medium" align="start">
+      <Card elevation="large" width="medium">
+        <CardBody height="small">
+          <Image
+            fit="cover"
+            src="//v2.grommet.io/assets/IMG_4245.jpg"
+            a11yTitle="bridge"
+          />
+        </CardBody>
+        <Box pad={{ horizontal: 'medium' }} responsive={false}>
```

Comment:
Why isn't this inside CardBody?

Reply:
Good question, I read about the [body tag best practice in HTML](https://stackoverflow.com/questions/2035462/multiple-htmlbody-html-body-in-same-file), and its recommended to use only one body tag, I know this is not html but since we are using html tags concept (header, body, footer) it might be confusing to contradict that with multiple body sections. 
I'm thinking we should rename CardBody to CardFlex or CardContent or something along those lines... thoughts?

Reply:
I think there should still be one `CardBody`. I just think both children should be inside it.